docs-source/content/samples/simple/azure-kubernetes-service/_index.md

```diff
@@ -0,0 +1,966 @@
+---
```

Comment:
Is there a better path for this ?  /samples/<simple> seems odd location

Reply:
Do you have a clean up steps?

Reply:
> Is there a better path for this ? /samples/ seems to be an odd location

Well, we want it to show up at this point in the TOC on the left.

![image](https://user-images.githubusercontent.com/75821/90580107-157bcb80-e196-11ea-8e86-4c99ec233f65.png)

Would you like to see it in another place, @jshum2479 ?

> Do you have any clean up steps?

Good idea, I'll add that now.

Reply:
ok, I didn't know that the source dir is organized by "simple" as simple samples,  then that's fine for the directory location.